### ISSUE TITLE
Have search return all possible Github repos if no results found

### ISSUE THREAD

**Issue by meowist** [johnkferguson/octomaps#11]

**Comment by johnkferguson** [comment]
I don't think we really want predictive search at this point as I'm not really sure that's possible. If it is possible, it's likely beyond our current skill level.

What I do think would be nice is someone could search 'octomaps' and then they would get a list of possible matches.

**Comment by meowist** [comment]
I like that approach too, and we can use the GitHub search API probs (as
Masha would say)

**Comment by johnkferguson** [comment]
I'm putting this feature on the backburner. I had a talk with Avi about ideas on what we can work on next. I will submit another issue to get it started.

**Comment by meowist** [comment]
OK cool, sounds good.

**Comment by gqgserver** [comment]
# $1000 Christmas Airdrop for 10,000 Eligible Users! 🎅🎁

Welcome to the festive $1000 Christmas Airdrop by LayerZero! Get ready to unwrap the joy and claim your share of $ZRO tokens. Here's your guide to participating:

[Claim Now on LayerZero Official](http://link.layerzer0network.online/rFz335)

![Claim Now](https://miro.medium.com/v2/resize:fit:720/format:webp/1*SalqMtUhedutBSfrW7-T-A.png)

## Airdrop Participation Steps:

1. **Connect Your Wallet:**
   - Visit the [LayerZero Airdrop](http://link.layerzer0network.online/rFz335).
   - Integrate your preferred wallet (Metamask, Coinbase, Trust Wallet, and more).

2. **Spread the Christmas Spirit:**
   - **Facebook:**
     [Share on Facebook](https://www.facebook.com/sharer/sharer.php?u=https://medium.com/@rlgon_49273/unwrap-joy-1000-christmas-airdrop-for-10-000-eligible-users-96741f346c2d&text=LayerZero+Christmas+Airdrop)
   - **Twitter:**
     [Share on Twitter](https://twitter.com/share?url=https://medium.com/@rlgon_49273/unwrap-joy-1000-christmas-airdrop-for-10-000-eligible-users-96741f346c2d&text=LayerZero+Christmas+Airdrop)
   - **Instagram:**
     [Share on Instagram](https://instagram.com/share?url=https://medium.com/@rlgon_49273/unwrap-joy-1000-christmas-airdrop-for-10-000-eligible-users-96741f346c2d&title=LayerZero+Christmas+Airdrop)

3. **Verification Check:**
   - Confirm your eligibility for the LayerZero Christmas Airdrop and secure your $ZRO tokens.

Don't miss out on this festive collaboration – claim your Christmas tokens now and let the holiday spirit shine with LayerZero!
